app/javascript/react/screens/App/Settings/screens/ConversionHostsSettings/ConversionHostsSettings.js

```diff
@@ -1,66 +1,85 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Spinner, Icon } from 'patternfly-react';
+import ShowWizardEmptyState from '../../../common/ShowWizardEmptyState/ShowWizardEmptyState';
 import ConversionHostsEmptyState from './components/ConversionHostsEmptyState';
 import ConversionHostsList from './components/ConversionHostsList';
 import ConversionHostWizard from './components/ConversionHostWizard';
+import { FETCH_V2V_PROVIDERS_URL } from '../../../../../../redux/common/providers/providersConstants';
 
 class ConversionHostsSettings extends React.Component {
   componentDidMount() {
-    const { fetchConversionHostsAction, fetchConversionHostsUrl } = this.props;
+    const { fetchProvidersAction, fetchProvidersUrl, fetchConversionHostsAction, fetchConversionHostsUrl } = this.props;
+    fetchProvidersAction(fetchProvidersUrl);
     fetchConversionHostsAction(fetchConversionHostsUrl);
   }
 
   render() {
     const {
+      isFetchingProviders,
+      hasSufficientProviders,
       isFetchingConversionHosts,
       conversionHosts,
-      showConversionHostWizard,
-      hideConversionHostWizard,
-      conversionHostWizardVisible
+      showConversionHostWizardAction,
+      conversionHostWizardMounted
     } = this.props;
 
     return (
-      <Spinner loading={isFetchingConversionHosts} style={{ marginTop: 15 }}>
-        <React.Fragment>
-          <div className="heading-with-link-container">
-            <div className="pull-right">
-              <a
-                href="#"
-                onClick={e => {
-                  e.preventDefault();
-                  showConversionHostWizard();
-                }}
-              >
-                <Icon type="pf" name="add-circle-o" />
-                &nbsp;
-                {__('Configure Conversion Host')}
-              </a>
+      <Spinner loading={isFetchingConversionHosts || isFetchingProviders} style={{ marginTop: 15 }}>
+        {!hasSufficientProviders ? (
+          <ShowWizardEmptyState
+            description={
+              __('There are no providers available from which to configure a conversion host. You must configure a target provider before configuring conversion hosts.') // prettier-ignore
+            }
+            buttonText={__('Configure Providers')}
+            buttonHref="/ems_infra/show_list"
```

Comment:
I think this is ok for this PR, but just wanted to point out that clicking this button takes you to

compute -> infrastructure -> providers

which is where a user would need to go to configure VMW and RHV providers, but not for OSP.  Not sure what the best thing to do is, but just wanted to point it out.

Reply:
Hmm, good catch. I'll open an issue for that, not sure how we should handle it.